distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -318,17 +318,17 @@ is_a value typ = if is_same_object value typ then True else
     if typ == Any then True else
         if is_error value then typ == Base.Error else
             case value of
-                Vector.Vector -> typ.is_same_object_as Vector.Vector
-                Array -> typ.is_same_object_as Array
-                Boolean -> if typ == Boolean then True else value == typ
-                Text -> typ == Text
-                Number -> if typ == Number then True else case value of
-                    Integer -> typ == Integer
-                    Decimal -> typ == Decimal
-                Date_Time.Date_Time -> typ.is_same_object_as Date_Time.Date_Time
-                Date.Date      -> typ.is_same_object_as Date.Date
-                Time_Of_Day.Time_Of_Day -> typ.is_same_object_as Time_Of_Day.Time_Of_Day
-                Time_Zone.Time_Zone -> typ.is_same_object_as Time_Zone.Time_Zone
+                _ : Vector.Vector -> typ.is_same_object_as Vector.Vector
+                _ : Array -> typ.is_same_object_as Base.Array
```

Comment:
Yes, this is a bit of a bummer. Unfortunately in this and other cases without `Base` we end up comparing `Array` vs `UnresolvedSymbol<Array>`. This looks like a separate issue.

Reply:
This situation started to bother me because unresolved symbol shouldn't leak that way. So i started digging.
Turned out that this was a bug in alias analysis where type usage would later affect global usage inside the pattern.
This is now fixed. Thanks for bringing it up @jdunkerley!